.travis.yml

```diff
@@ -44,6 +44,16 @@ matrix:
       services: docker
       before_install:
       addons:
+    - sudo: required
+      env: TOXENV=le_auto_wheezy
+      services: docker
+      before_install:
+      addons:
+    - sudo: required
```

Comment:
After chatting about this with people around the EFF office, we decided we only want these tests for distros where we recommend `certbot-auto` on certbot.eff.org. The reason for this is some devs often find themselves waiting around for Travis to finish so we only want to add jobs for distros where we are actually telling users to run `certbot-auto`.

With this in mind, the Jessie test and the associated Dockerfile should be deleted.